Apsitvarkom.DataAccess/PollutedLocationContext.cs

```diff
@@ -6,16 +6,18 @@ namespace Apsitvarkom.DataAccess;
 public interface IPollutedLocationContext
 {
     DbSet<PollutedLocation> PollutedLocations { get; }
+    DbSet<TidyingEvent> TidyingEvents { get; }
     DbContext Instance { get; }
 }
 
 public class PollutedLocationContext : DbContext, IPollutedLocationContext
 {
     public DbSet<PollutedLocation> PollutedLocations { get; private set; } = null!;
+    public DbSet<TidyingEvent> TidyingEvents { get; private set; } = null!;
     public DbContext Instance => this;
 
     public PollutedLocationContext(DbContextOptions<PollutedLocationContext> options) : base(options) { }
-
+    
     protected override void OnModelCreating(ModelBuilder modelBuilder)
```

Comment:
Didn't figure out a way to not allow creation of `TidyingEvent` if there is no such `PollutedLocation` yet.
Probably one way to achieve it could be introducing a `PollutedLocation` property inside of the `TidyingEvent` model and then specifying `modelBuilder.Entity<TidyingEvent>.Property(x => x.PollutedLocation).IsRequired();`

Reply:
I've already tried 
`modelBuilder.Entity<TidyingEvent>().HasOne<PollutedLocation>().WithMany(l => l.Events).HasForeignKey(e => e.PollutedLocationId).IsRequired();`

Reply:
another workaround could be just checking if a PollutedLocation by the given id exists in controller, I suppose

Reply:
> Probably one way to achieve it could be introducing a PollutedLocation property inside of the TidyingEvent model

And that's exactly what you should do IMO, as per [Fully defined relationships](https://learn.microsoft.com/en-us/ef/core/modeling/relationships?tabs=fluent-api%2Cfluent-api-simple-key%2Csimple-key#fully-defined-relationships) docs, they have the same exact setup: "Blog" is our "PollutedLocation" and "Post" is our "TidyingEvent". We have a 1 (polluted location) to N (many TidyingEvents) relation